sentences/fa/_common.yaml

```
@@ -0,0 +1,92 @@
+language: fa
+responses:
+  errors:
+    no_intent: "ببخشید نتونستم بفهمم"
+    no_area: "وجود ندارد {{ area }} هیچ منطقه ای به نام"
+    no_domain: "نیست {{ domain }} دارای {{ area }}"
+    no_device_class: "نیست {{ device_class }} دارای {{ area }}"
+    no_entity: "وجود ندارد {{ entity }} هیچ دستگاه یا موجودی به نام"
+    handle_error: "هنگام اجرای هدف یک خطای غیر منتظره بوجود آمد "
+lists:
+  color:
+    values:
+      # white
+      - in: "سفید"
+        out: "white"
+      - in: "white"
+        out: "white"
```

Why are you supporting English too?

Also, if values are the same, you can drop the `in`, `out` keys
```suggestion
      - "white"
```

---

Changed.